shared/core.js

```diff
@@ -40,7 +40,6 @@ module.exports = {
     'no-cond-assign': 'warn',
     'no-const-assign': 'error',
     'no-constant-condition': 'warn',
-    'no-control-regex': 'warn',
```

Comment:
Can you keep this line and change "warn" to "off"? We do this in other places in the linter config so it's more explicit + if eslint changes the defaults, we're not affected.